spec/factories/task.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :task do
+    title 'test task'
```

Comment:
let's make it a sequence with id in title

`sequence :title { |n| "task #{n} title" }`